packages/components/src/GuidedTour/GuidedTour.scss

```diff
@@ -1,22 +1,50 @@
 $tc-guidedtour-accent-color: $scooter !default;
 $tc-guidedtour-tooltip-padding: $padding-large !default;
 $tc-guidedtour-mask-opactiy: 0.25 !default;
-$tc-guidedtour-close-button-size: $svg-sm-size !default;
+$tc-guidedtour-close-button-size: $svg-md-size !default;
+$tc-guidedtour-nav-button-size: $svg-sm-size !default;
+$tc-guidedtour-controls-color: $dove-gray !default;
 
 :global(#___reactour) {
 	.tc-guided-tour {
 		--reactour-accent: #{$tc-guidedtour-accent-color};
-		padding: $tc-guidedtour-tooltip-padding (3 * $tc-guidedtour-tooltip-padding) $tc-guidedtour-tooltip-padding $tc-guidedtour-tooltip-padding;
+		padding: $tc-guidedtour-tooltip-padding (2 * $tc-guidedtour-tooltip-padding) $tc-guidedtour-tooltip-padding $tc-guidedtour-tooltip-padding;
 		border-color: rgb(226, 226, 226);
 		box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.2);
 
 		[data-tour-elem='controls'] {
-			margin: 0 (-2 * $tc-guidedtour-tooltip-padding) 0 0;
+			margin: 0 (-1 * $tc-guidedtour-tooltip-padding) 0 0;
 			justify-content: space-between;
 
+			> button,
 			+ button {
-				height: $tc-guidedtour-close-button-size;
-				width: $tc-guidedtour-close-button-size;
+				color: $tc-guidedtour-controls-color;
```

Comment:
Perhaps you can inline that in both `> button` and `+ button` selectors, as they exist, and the common aspect is held by the sass var.
It would ease a little the readability, by having 2 selectors instead of 4 for the 2 elements

Reply:
I would not duplicate properties